app/controllers/Api/V1/SongsController.rb

```diff
@@ -0,0 +1,59 @@
+class Api::V1::SongsController < ApplicationController
+  def index
```

Comment:
Great work on this beast controller. Still opportunity to refactor with facades, services, and serializers but this is functioning and appears to be handling errors.